udata/core/contact_point/models.py

```diff
@@ -9,6 +9,14 @@
     "contact": _("Contact"),
     "creator": _("Creator"),
     "publisher": _("Publisher"),
+    "rightsHolder": _("RightsHolder"),
+    "custodian": _("Custodian"),
+    "distributor": _("Distributor"),
+    "originator": _("Originator"),
+    "principalInvestigator": _("PrincipalInvestigator"),
```

Comment:
I would add a space in the different contact roles labels?
```suggestion
    "principalInvestigator": _("Principal Investigator"),
```